build-tools/tasks/docs.js

```diff
@@ -15,27 +14,11 @@ module.exports = function docs() {
       TagEditor: ['getTagsDiff'],
     },
   });
-  testUtilDocs();
+  writeTestUtilsDocumentation({
+    outDir: path.join(workspace.apiDocsPath, 'test-utils-doc'),
+    tsconfigPath: require.resolve('../../src/test-utils/tsconfig.json'),
+    domUtilsRoot: 'src/test-utils/dom/index.ts',
+    selectorsUtilsRoot: 'src/test-utils/selectors/index.ts',
+  });
   return Promise.resolve();
 };
-
-function testUtilDocs() {
-  ['dom', 'selectors'].forEach(testUtilType => {
-    const baseWrapperDefinitions = require(`@cloudscape-design/test-utils-core/test-utils-doc/${testUtilType}`);
```

Comment:
NB: in the old version we used to mix the base classes into this documentation. it should not be happening anymore

The consuming code will import them explicitly